README.md

```diff
@@ -1,2 +1,55 @@
-# the-experts
-An app for connecting contractors with users
+##### the-experts
+
+## [ OUR HOSTED Experts {Not Working For NOW} ](https://the-experts.herokuapp.com/)
+
+## Local installation
+```
+ git clone https://github.com/FACN7/the-experts.git
+ cd the-experts
+ npm i 
+```
+In the project directory AND in the the-experts-fe directory, you have to run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+## User journey
+Well, it's and app for connecting contractors with users, you can enter to the app and search for a contractor
```

Comment:
Here you can put the User Journey we created - 'Allow a user to find a suitable contractor'. Also the User Stories we created:

- As a User, I want to search for a contractor (based on job title) and see a list of results
- As a User, I want to come to homepage and understand what the website is for